### PR TITLE
[threat actors] Fix threat actors related to Lotus Panda

### PR DESCRIPTION
**Pull request by Mathieu4141**
There were a few conflicting aliases around Naikon/Lotus Panda:
image

Here are my the changes I suggest to fix the conflicts:
 - Remove the `Lotus Panda` threat actor. `Lotus Panda` is an alias of `Naikon`
 - Remove `Lotus Panda` alias from `Thrip`
 - Remove `Hellsing` alias from `Naikon`
 - Remove `APT30` alias from `Naikon`
 - Remove `Thrip` alias from `Stealth Mango and Tangelo`

### CODE REVIEW

**Comment by r0ny123** [comment]
Thanks for the PR, @Mathieu4141. Good to see `Thrip` and `Stealth Mango and Tangelo` branched out. I have a couple of suggestions for AKAs

- `APT30(G0013)`/ `Naikon(G0019)`/ `Camerashy`/ `OVERRIDE PANDA`/ `BRONZE GENEVA`/ `BRONZE STERLING`/ `Unit 78020`
- Remove the `Esile` https://github.com/MISP/misp-galaxy/blob/9c6f106928d2dbaa695beeae7fafc09baaec6049/clusters/threat-actor.json#L893 from the synonym of `Lotus Blossom` as it is malware used by them and add `LOTUS PANDA` as aka.
- Also branch out `GOBLIN PANDA`/ `Cycldek`/ `Conimes` from Hellsing.

Please address these and we are good to go.

**Comment by adulau** [comment]
For `KDR_ESILE` and `Esile`, I would add it in Tools https://github.com/MISP/misp-galaxy/blob/main/clusters/tool.json and make a relationship between the two with a `used-by`.

File: clusters/threat-actor.json [hunk 1]
```diff
@@ -6456,10 +6448,12 @@
...
           "https://www.symantec.com/blogs/threat-intelligence/thrip-hits-satellite-telecoms-defense-targets",
           "https://attack.mitre.org/groups/G0076/",
-          "https://go.crowdstrike.com/rs/281-OBQ-266/images/Report2020CrowdStrikeGlobalThreatReport.pdf"
+          "https://go.crowdstrike.com/rs/281-OBQ-266/images/Report2020CrowdStrikeGlobalThreatReport.pdf",
+          "https://cyberthreat.thalesgroup.com/sites/default/files/2022-05/THALES%20THREAT%20HANDBOOK%202022%20Light%20Version_1.pdf"
         ],
         "synonyms": [
-          "G0076"
+          "G0076",
+          "UTK78"
```
Comment (r0ny123) [inline clusters/threat-actor.json:6456]:
```suggestion
          "ATK78"
```

**Review by r0ny123 — CHANGES_REQUESTED** [review]

File: clusters/threat-actor.json [hunk 2]
```diff
@@ -10039,6 +9983,41 @@
...
+          "https://www.crowdstrike.com/blog/meet-crowdstrikes-adversary-of-the-month-for-august-goblin-panda/",
+          "https://securelist.com/cycldek-bridging-the-air-gap/97157/",
+          "https://www.fortinet.com/blog/threat-research/cta-security-playbook--goblin-panda.html",
+          "https://go.crowdstrike.com/rs/281-OBQ-266/images/Report2020CrowdStrikeGlobalThreatReport.pdf",
+          "https://cyberthreat.thalesgroup.com/sites/default/files/2022-05/THALES%20THREAT%20HANDBOOK%202022%20Light%20Version_1.pdf"
+        ],
+        "synonyms": [
+          "Conimes",
+          "Cycldek",
+          "ATK78"
```
Comment (r0ny123) [inline clusters/threat-actor.json:10016]:
```suggestion

```

**Review by r0ny123 — COMMENTED** [review]
Remove `ATK78` from `GOBLIN PANDA`

File: clusters/threat-actor.json [hunk 3]
```diff
@@ -9872,7 +9816,42 @@
...
+          "https://www.crowdstrike.com/blog/meet-crowdstrikes-adversary-of-the-month-for-august-goblin-panda/",
+          "https://securelist.com/cycldek-bridging-the-air-gap/97157/",
+          "https://www.fortinet.com/blog/threat-research/cta-security-playbook--goblin-panda.html",
+          "https://go.crowdstrike.com/rs/281-OBQ-266/images/Report2020CrowdStrikeGlobalThreatReport.pdf",
+          "https://cyberthreat.thalesgroup.com/sites/default/files/2022-05/THALES%20THREAT%20HANDBOOK%202022%20Light%20Version_1.pdf"
+        ],
+        "synonyms": [
+          "Conimes",
+          "Cycldek",
+          "ATK34"
```
Comment (r0ny123) [inline clusters/threat-actor.json:9849]: ATK34 overlaps with APT30, not GOBLIN PANDA.
See 1st bullet of https://github.com/MISP/misp-galaxy/pull/749#issuecomment-1217452683

Comment (Mathieu4141) [reply clusters/threat-actor.json:9849]: Thales says it overlaps with Golbin Panda page 53 here: https://cyberthreat.thalesgroup.com/sites/default/files/2022-05/THALES%20THREAT%20HANDBOOK%202022%20Light%20Version_1.pdf
Screen Shot 2022-08-17 at 1 53 37 PM

Comment (r0ny123) [reply clusters/threat-actor.json:9849]: On another page, they say it is APT30. Weird

Comment (Mathieu4141) [reply clusters/threat-actor.json:9849]: Oh
I'll just remove that alias then, is it ok?

Comment (r0ny123) [reply clusters/threat-actor.json]: For now ok. Let me ask someone from Thales.

**Comment by r0ny123** [comment]
So, now just merge the APT30 and Naikon with the following synonyms mentioned here https://github.com/MISP/misp-galaxy/pull/749#issuecomment-1217452683.

File: clusters/threat-actor.json [hunk 4]
```diff
@@ -805,21 +805,20 @@
...
+          "https://cluster25.io/2022/04/29/lotus-panda-awake-last-strike/"
         ],
         "synonyms": [
           "PLA Unit 78020",
-          "APT 30",
-          "APT30",
           "Override Panda",
           "Camerashy",
-          "APT.Naikon",
           "Lotus Panda",
```
Comment (r0ny123) [inline clusters/threat-actor.json:821]:
```suggestion
```

Comment (r0ny123) [reply clusters/threat-actor.json:821]: Remove `Lotus Panda`.

File: clusters/threat-actor.json [hunk 5]
```diff
@@ -891,7 +890,6 @@
         "synonyms": [
           "Spring Dragon",
           "ST Group",
-          "Esile",
           "DRAGONFISH",
```
Comment (r0ny123) [inline clusters/threat-actor.json:893]:
```suggestion
          "DRAGONFISH",
          "LOTUS PANDA",
```

Comment (r0ny123) [reply clusters/threat-actor.json:893]: Add `LOTUS PANDA` here

**Comment by Mathieu4141** [comment]
For the alias `Lotus Panda`, are you sure about moving it from `Naikon` to `Lotus Blossum`? Most reports I find associate `Lotus Panda` with `Naikon`:
https://www.cfr.org/cyber-operations/apt-30
https://cluster25.io/2022/04/29/lotus-panda-awake-last-strike/
https://blog.koddos.net/chinese-lotus-panda-hackers-resurface-with-more-sophisticated-attacks/

**Comment by r0ny123** [comment]
CFR is just sourcing from other reports, and koddos (never heard of) is a regular news site. Cluster25's report is on APT30/Naikon. I don't see any `LOTUS PANDA` activity. They fall short in attribution.

File: clusters/threat-actor.json [hunk 6]
```diff
@@ -805,23 +805,28 @@
...
-          "APT30",
           "Override Panda",
           "Camerashy",
-          "APT.Naikon",
           "Lotus Panda",
-          "Hellsing",
           "BRONZE GENEVA",
-          "G0019"
+          "G0019",
+          "APT 30",
```
Comment (r0ny123) [inline clusters/threat-actor.json:827]:
```suggestion
          "APT30",
          "BRONZE STERLING",
```

**Review by r0ny123 — CHANGES_REQUESTED** [review]

File: clusters/threat-actor.json [hunk 7]
```diff
@@ -891,12 +889,12 @@
...
           "Spring Dragon",
           "ST Group",
-          "Esile",
           "DRAGONFISH",
           "BRONZE ELGIN",
           "ATK1",
           "G0030",
-          "Red Salamander"
+          "Red Salamander",
+          "Lotus Panda"
```
Comment (r0ny123) [inline clusters/threat-actor.json:897]:
```suggestion
          "LOTUS PANDA"
```

File: clusters/threat-actor.json [hunk 8]
```diff
@@ -805,23 +805,28 @@
...
+          "https://www.ptsecurity.com/ww-en/analytics/pt-esc-threat-intelligence/eagle-eye-is-back-apt30/",
+          "https://www2.fireeye.com/rs/fireye/images/rpt-apt30.pdf",
+          "https://attack.mitre.org/wiki/Group/G0013",
+          "https://www.mandiant.com/sites/default/files/2021-09/rpt-apt30.pdf"
         ],
         "synonyms": [
           "PLA Unit 78020",
-          "APT 30",
-          "APT30",
           "Override Panda",
```
Comment (r0ny123) [inline clusters/threat-actor.json:823]:
```suggestion
          "OVERRIDE PANDA",
```

**Review by r0ny123 — CHANGES_REQUESTED** [review]

**Comment by r0ny123** [comment]
LGTM! 🚀

File: clusters/threat-actor.json [hunk 9]
```diff
@@ -805,23 +805,28 @@
           "https://www.fireeye.com/blog/threat-research/2014/03/spear-phishing-the-news-cycle-apt-actors-leverage-interest-in-the-disappearance-of-malaysian-flight-mh-370.html",
           "https://www.cfr.org/interactive/cyber-operations/apt-30",
           "https://media.kasperskycontenthub.com/wp-content/uploads/sites/43/2018/03/07205555/TheNaikonAPT-MsnMM1.pdf",
+          "https://usa.kaspersky.com/resource-center/threats/naikon-targeted-attacks",
           "https://blog.trendmicro.com/trendlabs-security-intelligence/bkdr_rarstone-new-rat-to-watch-out-for/",
           "https://securelist.com/the-chronicles-of-the-hellsing-apt-the-empire-strikes-back/69567/",
```
Comment (r0ny123) [inline clusters/threat-actor.json:810]:
```suggestion
```

File: clusters/threat-actor.json [hunk 10]
```diff
@@ -1185,15 +1139,7 @@
...
-          "https://www.cfr.org/interactive/cyber-operations/hellsing",
-          "https://www.crowdstrike.com/blog/meet-crowdstrikes-adversary-of-the-month-for-august-goblin-panda/",
-          "https://securelist.com/cycldek-bridging-the-air-gap/97157/",
-          "https://www.fortinet.com/blog/threat-research/cta-security-playbook--goblin-panda.html"
-        ],
-        "synonyms": [
-          "Goblin Panda",
-          "Conimes",
-          "Cycldek"
+          "https://www.cfr.org/interactive/cyber-operations/hellsing"
```
Comment (r0ny123) [inline clusters/threat-actor.json:1142]:
```suggestion
          "https://www.cfr.org/interactive/cyber-operations/hellsing,"
          "https://securelist.com/the-chronicles-of-the-hellsing-apt-the-empire-strikes-back/69567"
```

**Review by r0ny123 — CHANGES_REQUESTED** [review]

File: clusters/threat-actor.json (hunk 9 above)
Comment (r0ny123) [inline clusters/threat-actor.json:810]:
```suggestion
```

**Review by r0ny123 — CHANGES_REQUESTED** [review]

**Comment by r0ny123** [comment]
LGTM! 🚀 
Thanks for all the work @Mathieu4141!

**Comment by Mathieu4141** [comment]
Thanks for your review and help on this!

**Comment by adulau** [comment]
Looks very good. Thank to you both for the work and reviews.

**Pull request merged**